src/pages/issue/read/components/Sidebar.tsx

```diff
@@ -25,7 +26,7 @@ export default function Sidebar() {
           selectedIssue.assignees.map((assignee) => (
             <div className="flex items-center gap-1 text-gray-600">
               <PiDotDuotone className="min-w-[24px] h-6 !text-yellow-500" />
-              <span className="truncate">{assignee}</span>
+              <span className="truncate">{shortenAddress(assignee, 6)}</span>
```

Comment:
Does it have a way to see the full address? Like a tooltip when you hover for a few seconds?